src/Digdir.Domain.Dialogporten.Janitor/Program.cs

```diff
@@ -43,7 +44,12 @@ static void BuildAndRun(string[] args)
     // a command is the scope of the application.   
     builder.Host.UseDefaultServiceProvider(options => options.ValidateScopes = false);
 
-    builder.Configuration.AddUserSecrets<Program>();
+    builder.Configuration
+        .AddJsonFile("appsettings.json", optional: true, reloadOnChange: true)
+        .AddJsonFile($"appsettings.{builder.Environment.EnvironmentName}.json", optional: true, reloadOnChange: true)
+        .AddUserSecrets<Program>()
+        .AddEnvironmentVariables();
+
```

Comment:
Dette trengs ikke. Cocona laster alt sett bort i fra user secrets hvilket må legges til manuelt som jeg gjorde. Har testet at det også skal legges til i korrekt rekkefølge (samme som over). Eneste er at en console app bruker DOTNET_ENVIRONMENT og ikke ASPNETCORE_ENVIRONMENT som du har fiksa i IaC filene 🙌

Reply:
Btw @arealmaas - du kan bruke DOTNET_ENVIRONMENT fremfor ASPNETCORE_ENVIRONMENT for å sette miljønavn på generelt grunnlag. DOTNET_ENVIRONMENT tar meg bekjent alltid presidents over ASPNETCORE_ENVIRONMENT, selv i en asp.net applikasjon.

Reply:
nice nice!